parser/openfoodfacts_taxonomy_parser/parser.py

```diff
@@ -418,27 +428,32 @@ def delete_used_properties(self):
         query = "MATCH (n) SET n.is_before = null, n.parents = null"
         self.session.run(query)
 
-    def create_fulltext_index(self):
-        query = """
-            CREATE FULLTEXT INDEX nodeSearchIds FOR (n:ENTRY) ON EACH [n.id]
-            OPTIONS {indexConfig: {`fulltext.analyzer`: 'keyword'}}
-        """
-        self.session.run(query)
+    def create_fulltext_index(self, multi_label, branch_name):
+        """Create indexes for search"""
+        index_name = multi_label.replace(":", "_")
+        query = [
+            f"""CREATE FULLTEXT INDEX {index_name+'_SearchIds'}
+            FOR (n:{'b_'+branch_name}) ON EACH [n.id]\n"""
```

Comment:
Why not use multilabel here ?

Reply:
It seems multiple labels are not allowed for indexes, hence I restricted the label to the branch name.

Reply:
hum that's really sad … I think thus we should change our strategy…

That is add three labels on entries at import time: branch_name, taxonomy_name and a blend of both.
The blend would be g_<branch_name>_<taxonomy_name> (prefix g like graph, but you may have a better idea !).

So throughout the code, you will use g_<branch_name>_<taxonomy_name>
But I would like to keep the b_ and t_ label, for later (when we would allow editing more than one taxonomy in the same project).

Are you ok to do this refactor ? Do you want to do it in a second phase (merge that and do it in a new PR ?)

Reply:
I think I'll do this refactor over here, and I'll simultaneously also give the frontend multilabel PR (it's almost complete). So, when both PRs are ready, they can be merged together. Does this sound good?